homeassistant/components/google_generative_ai_conversation/__init__.py

```diff
@@ -265,6 +267,15 @@ async def async_migrate_integration(hass: HomeAssistant) -> None:
                     unique_id=None,
                 ),
             )
+            hass.config_entries.async_add_subentry(
```

Comment:
Should you remove this block here and line 322? This will be taken care by async_migrate_entry and avoid duplicate code.
At least that's what I do in #147563